packages/nextjs/package.json

```diff
@@ -47,10 +47,10 @@
     "autoprefixer": "^10.4.12",
     "eslint": "^8.15.0",
     "eslint-config-next": "^14.0.4",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-prettier": "^5.1.3",
```

Comment:
Had to update this because `yarn next:lint` was failing : 
![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/e6587a26-31d2-44d5-8339-249a33c91ad2)